src/ui/Omnibox/Omnibox.ts

```diff
@@ -690,17 +692,25 @@ export class Omnibox extends Component {
   }
 
   private handleTabPressForOldOmniboxAddon() {
-    if (this.lastSuggestions && this.lastSuggestions[0] && this.lastSuggestions[0].dom) {
-      const firstSelected = $$(this.lastSuggestions[0].dom).find('.coveo-omnibox-selected');
-      const firstSelectable = $$(this.lastSuggestions[0].dom).find('.coveo-omnibox-selectable');
-      if (firstSelected) {
-        $$(firstSelected).trigger('tabSelect');
-      } else if (firstSelectable) {
-        $$(firstSelectable).trigger('tabSelect');
+    const domSuggestions = this.lastSuggestions.filter(suggestions => suggestions.dom).map(suggestions => $$(suggestions.dom));
+    const selected = this.findAllElementsWithClass(domSuggestions, '.coveo-omnibox-selected');
```

Comment:
`$$(domSuggestions).findAll('.coveo-omnibox-selected')`

Reply:
`domSuggestions` is an array